### ISSUE TITLE
Team reflection - October 9th

### ISSUE THREAD

**Issue by nebskers** [jacob-barna/TripleJR#30]
This is where we are going to comment on our reflection.  Please comment on any issues that you had and any recommendations you would like to see happen.

**Comment by nebskers** [comment]
I am not okay with Ron laughing at me.

**Comment by cyb3rc0wb0y** [comment]
During the development of our proposal and use/mis-use case development, GitHub collaboration was not used to the full potential or was slow to start off. However, for the assurance assignment there was better collaboration on GitHub and better use of the features.

**Comment by nebskers** [comment]
The slow start was due to all the communication via Discord for most of the collabration.  Everyone quickly was open and willing to take on tasks individually assigned.  We collaborated very well on the tasks that each person would be assigned.

**Comment by cyb3rc0wb0y** [comment]
During the assurance assignment, the creation of claims and rebuttals were underestimated and led to problems coming up with broad claims. However, after getting started with a strong initial claim the team was able to develop the remaining claims with less trouble.

**Comment by cyb3rc0wb0y** [comment]
The initial thought that claims had to be proved "true" turned out to be false and with evidence the claims could be proved "false" or not secure.

**Comment by ramirezronald** [comment]
We also will be using the professors suggestions he mentioned, by utilizing the Github issue board to make comments on issues we are working on. This helps our team stay organized.

**Comment by cyb3rc0wb0y** [comment]
Collaboration from other sources could be moved from applications such as Discord and integrated into GitHub to better track communication and collaboration.

**Comment by nebskers** [comment]
There was one class where we assumed we would have group time but there wasn't any.  I think we all relied on that to communicate further issues.  Therefore, we all need to communicate on github or discord when we have questions/concerns.

**Comment by cyb3rc0wb0y** [comment]
@ramirezronald you make a good point with the organization.

**Comment by nebskers** [comment]
Please have all your reflections in by Friday morning and I will create a summary for the assignment.

**Comment by ramirezronald** [comment]
@cyb3rc0wb0y thank you for the kind words !

**Comment by cyb3rc0wb0y** [comment]
cool cool

**Comment by ramirezronald** [comment]
thank you for doing that @nebskers  ! ! !

**Comment by jacob-barna** [comment]
Things that went well: 
Team work - I saw everyone do their part and collaborate when refining
Tools - lucid chart is cool ... easy to collaborate using this
Class time - it is nice to sit together and use class time to work on our project... it's easier than meeting up on the weekend

Things that didn't go well: 
???

Things we should try: 
Emailing the teacher to get feedback...

**Comment by nebskers** [comment]
I will write up the summary and have it by tomorrow noon.

**Comment by nebskers** [comment]
Here is the summary right now: 
As a team, we have collectively agreed that even though we have successfully completed all the project requirements as of now, we have had some ups and downs.  We use Lucid chart for our diagrams, Discord and GitHub, and class time for our communication and collaboration.   During the development of our proposal and use/misuse case development, GitHub collaboration was not used to the full potential or was slow to start. However, for the assurance assignment, there was better collaboration on GitHub and better use of the features. The slow start was due to all the communication via Discord. Collaboration from other sources could be moved from applications such as Discord and integrated into GitHub to track communication and collaboration better. During the assurance assignment, the creation of claims and rebuttals were underestimated and led to problems coming up with broad claims. However, after getting started with a strong initial claim, the team was able to develop the remaining claims with less trouble. The initial thought that claims had to be proved "true" turned out to be false, and with evidence, the claims could be proved "false" or not secure. We also will be using the professor's suggestions he mentioned, by utilizing the Github issue board to make comments on issues we are working on. This helps our team stay organized. Regarding the individual contributions, we are in an agreeance that everyone has been open to receiving responsibilities and successful at meeting the deadlines.
